### PR TITLE
fix(a380x): add missing telex check

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #[issue_no]

## Summary of Changes

Show the PopUp to enable telex at the start of the flight.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Load the flight, make sure the popup occurs and asks to enable TELEX.
Select yes or no, start a new flight and make sure it does not appear again.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by Benjozork — APPROVED** [review]

File: fbw-a380x/src/systems/extras-host/modules/telex_check/TelexCheck.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// SPDX-License-Identifier: GPL-3.0
+
+import { NXDataStore, PopUpDialog } from '@flybywiresim/fbw-sdk';
+
+export class TelexCheck {
+  constructor() {
+    console.log('TelexCheck: Created');
+  }
+
+  public startPublish(): void {
```
Comment (tracernz) [inline fbw-a380x/src/systems/extras-host/modules/telex_check/TelexCheck.ts:13]: It's not a publisher so this seems weird.

Comment (Saschl) [reply fbw-a380x/src/systems/extras-host/modules/telex_check/TelexCheck.ts:13]: Yes 😄 will change it

**Comment by Benjozork** [comment]
Needs a changelog entry

**Comment by Benjozork** [comment]
Dev Team Testing Report

Object of testing : #9190
Aircraft          : A380X
Tier of Testing   : 1
Date              : 01/11/2024

Testing Process: 
- [x] Open aircraft after building -> popup shows up
- [x] Re-load aircraft  -> popup does not show up

Negatives:
- N/A

Testing Results: 
QA Passed

**Pull request merged**